### PR TITLE
Change doclayout pydata

### PR DESCRIPTION
**Pull request by agoscinski**
See commit messages for changes and look at documentation preview.

----
:books: Documentation preview :books:: https://scikit-matter--204.org.readthedocs.build/en/204/

### CODE REVIEW

File: docs/src/index.rst [hunk 1]
```diff
@@ -1,19 +1,88 @@
...
+                <div class="card-body flex-fill">
+                    <h5 class="card-title">
+
+
+.. only:: html
+
+  .. image:: /examples/selection/images/thumb/sphx_glr_FeatureSelection-WHODataset_thumb.png
+    :alt:
+
+  :ref:`getting_started-selection`
```
Comment (PicoCentauri) [inline docs/src/index.rst:24]: This is super nice these cards!

Can you maybe set the background to transparent. Dark mode user here. 😝

Also, can we maybe have the three cards in one line for widescreens and responsivly stacked. Seems like you are using bootstrap. I think there is an option. We can look together.

Comment (rosecers) [reply docs/src/index.rst:24]: Agreement on the transparency, also a dark mode user

Comment (agoscinski) [reply docs/src/index.rst]: added

**Comment by PicoCentauri** [comment]
The cards are very nice. I did some minor changes to the layout. I removed the shadows and let the cards stack a bit better on the desktop and phone

Screenshot 2023-08-15 at 11 33 19

Screenshot 2023-08-15 at 11 33 31

**Comment by agoscinski** [comment]
thanks for the frontend work, looks better. Would squash your commit with last commit 9ef24bde672a9de0d2c50ad95dac6223169f640d and then cen be merged by rebase from my side.

**Comment by PicoCentauri** [comment]
Yes, I am not 100% happy but I can not use another hour on this. 😅

I approve when you rebased.

**Review by PicoCentauri — APPROVED** [review]
Thanks again. I like the new design!

**Pull request merged**